test/handler-test.js

```diff
@@ -121,7 +121,7 @@ describe('handler', () => {
     expect(impressionJson.request_uuid).to.equal('req-uuid');
     expect(impressionJson.feeder_podcast).to.equal(1234);
     expect(impressionJson.feeder_episode).to.equal('1234-5678');
-    expect(impressionJson.listener_session).to.equal('listener-session-1');
+    expect(impressionJson.listener_session).to.equal(downloadJson.listener_session);
```

Comment:
No longer know what the listenerSession will be ... so just doing some basic checks on length/uniqueness in here.